Google.PowerShell.IntegrationTests/Logging/Logging.GcLogSink.Tests.ps1

```diff
@@ -72,3 +69,347 @@ Describe "Get-GcpsLogSink" {
+            $createdSink.EndTime | Should BeNullOrEmpty
+
+            $createdSink = Get-GcLogSink -Sink $sinkNameTwo
+            $createdSink | Should Not BeNullOrEmpty
+            $createdSink.Name | Should BeExactly $sinkNameTwo
+            $createdSink.Destination | Should BeExactly "storage.googleapis.com/$bucketTwo"
+            $createdSink.OutputVersionFormat | Should BeExactly V2
+            $createdSink.WriterIdentity | Should Not BeNullOrEmpty
+            $createdSink.Filter | Should BeNullOrEmpty
+            $createdSink.StartTime | Should BeNullOrEmpty
+            $createdSink.EndTime | Should BeNullOrEmpty
+        }
+        finally {
+            gcloud beta logging sinks delete $sinkName --quiet 2>$null
+            gcloud beta logging sinks delete $sinkNameTwo --quiet 2>$null
+        }
+    }
+
+    It "should work with -NoUniqueWriterIdentity" {
+        $r = Get-Random
+        $pubsubTopic = "gcloud-powershell-testing-pubsubtopic-$r"
+        $sinkName = "gcps-new-gclogsink-$r"
+        $sinkNameTwo = "gcps-new-gclogsink2-$r"
+        try {
+            New-GcLogSink $sinkName -PubSubTopicDestination $pubsubTopic -NoUniqueWriterIdentity
+            New-GcLogSink $sinkNameTwo -PubSubTopicDestination $pubsubTopic
+            Start-Sleep -Seconds 1
+
+            $createdSink = Get-GcLogSink -Sink $sinkName
+            $createdSink | Should Not BeNullOrEmpty
+            $createdSink.Name | Should BeExactly $sinkName
+            $createdSink.Destination | Should BeExactly "pubsub.googleapis.com/projects/$project/topics/$pubsubTopic"
+            $createdSink.OutputVersionFormat | Should BeExactly V2
+            $createdSink.WriterIdentity | Should BeExactly "serviceAccount:cloud-logs@system.gserviceaccount.com"
+            $createdSink.Filter | Should BeNullOrEmpty
+            $createdSink.StartTime | Should BeNullOrEmpty
+            $createdSink.EndTime | Should BeNullOrEmpty
+
+            $createdSink = Get-GcLogSink -Sink $sinkNameTwo
+            $createdSink | Should Not BeNullOrEmpty
+            $createdSink.Name | Should BeExactly $sinkNameTwo
+            $createdSink.Destination | Should BeExactly "pubsub.googleapis.com/projects/$project/topics/$pubsubTopic"
+            $createdSink.OutputVersionFormat | Should BeExactly V2
+            $createdSink.WriterIdentity | Should Not Be "serviceAccount:cloud-logs@system.gserviceaccount.com"
+            $createdSink.Filter | Should BeNullOrEmpty
+            $createdSink.StartTime | Should BeNullOrEmpty
+            $createdSink.EndTime | Should BeNullOrEmpty
+        }
+        finally {
+            gcloud beta logging sinks delete $sinkName --quiet 2>$null
+            gcloud beta logging sinks delete $sinkNameTwo --quiet 2>$null
+        }
+    }
+
+
+    It "should work with -Before and -After" {
+        $r = Get-Random
+        # These topics already has permission for cloud-logs to write to.
```

Comment:
wording, maybe:
These topics already have permission to write to cloud-logs?

and below